kubernetes/template-generator/template_generator.py

```diff
@@ -0,0 +1,358 @@
+#!/usr/bin/env python
+
+import re
+import yaml
+import json
+
+from container_os import *
+
+### TEMPLATE GENERATORS
+class SanitizeException(Exception):
+    pass
+
+class TemplateGenerator(object):
+    """
```

Comment:
https://github.com/leapp-to/guidelines/blob/master/python.md  https://pythonhosted.org/an_example_pypi_project/sphinx.html#function-definitions